src/main/resources/tags/sensitive_materials.json

```diff
@@ -0,0 +1,11 @@
+{
+	"values" : [
```

Comment:
Not sure but shouldn't buttons and redstone conponents be included too?

Reply:
There are lots of things that aren't included because they aren't used.